.zappr.yaml

```diff
@@ -0,0 +1,17 @@
+approvals:
+  minimum: 1
+  ignore: pr_opener
+  pattern: "^(👍|:\\+1:|:thumbsup:|:shipit:)$"
+  veto:
+    pattern: "^(👎|:-1:|:thumbsdown:)$"
+  from:
+    users:
```

Comment:
Good. I forgot that it's fine to have someone from not formally inside the team.